torchbenchmark/models/Super_SloMo/__init__.py

```diff
@@ -30,7 +41,7 @@ def __init__(self, device=None, jit=False):
         root = str(Path(__file__).parent)
         self.args = args = Namespace(**{
             'dataset_root': f'{root}/dataset',
-            'train_batch_size': 6,
+            'train_batch_size': train_bs,
```

Comment:
I think you can delete this line since there are no references to `self.args.train_batch_size`. Or modify the references below to use this, and potentially `self.args.eval_batch_size`

Reply:
Fixed. Thanks for pointing out!